android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -881,9 +882,25 @@ public void onFreeMeetingUpgradeToGiftFreeTrialStart() {}
   @Override
   public void onFreeMeetingNeedToUpgrade(FreeMeetingNeedUpgradeType type, String gifUrl) {}
   @Override
-  public void onUserAudioStatusChanged(long userId, AudioStatus audioStatus) {}
+  public void onUserAudioStatusChanged(long userId, AudioStatus audioStatus) {
+    InMeetingService inMeetingService = ZoomSDK.getInstance().getInMeetingService();
+
+    if (userId == inMeetingService.getMyUserID()) {
+      final InMeetingUserInfo userInfo = inMeetingService.getMyUserInfo();
+
+      sendEvent("MeetingEvent", "myAudioStatus", userInfo);
+    }
+  }
   @Override
-  public void onUserVideoStatusChanged(long userId, VideoStatus status) {}
+  public void onUserVideoStatusChanged(long userId, VideoStatus status) {
+    InMeetingService inMeetingService = ZoomSDK.getInstance().getInMeetingService();
+
+    if (userId == inMeetingService.getMyUserID()) {
+      final InMeetingUserInfo userInfo = inMeetingService.getMyUserInfo();
+
+      sendEvent("MeetingEvent", "myVideoStatus", userInfo);
```

Comment:
:thinking: I believe we should add a verb to the event name
```suggestion
      sendEvent("MeetingEvent", "myVideoStatusChanged", userInfo);
```

analogously for `myAudioStatus` -> `myAudioStatusChanged`